code/modules/power/apc.dm

```diff
@@ -415,13 +414,14 @@
 			update_icon()
 			updating_icon = 0
 
-/obj/machinery/power/apc/proc/spookify()
-	if(spooky) return // Fuck you we're already spooky
-	spooky=1
-	update_icon()
-	spawn(10)
-		spooky=0
-		update_icon()
+/obj/machinery/power/apc/get_spooked()
+	if(opened || wiresexposed)
+		return
+	if(stat & (NOPOWER | BROKEN))
+		return
+	addtimer(CALLBACK(src, .proc/update_icon, TRUE), 10)
+	overlays.Cut()

```

Comment:
cut_overlays?

Reply:
Ahh yes, I'll update this to use that

Reply:
Okay, well I made this use `cut_overlays()` instead of `overlays.Cut()` - there seems to be a delay in the former, which means the result looks slightly unwieldy - but I suppose it's better to use the subsystem rather than just do it the old-fashioned way